clients/tabby-chat-panel/src/index.ts

```diff
@@ -219,8 +220,11 @@ export interface ServerApi {
    */
   executeCommand: (command: ChatCommand) => Promise<void>
 
+  // @deprecated
   showError: (error: ErrorMessage) => void
+  // @deprecated
   cleanError: () => void
+
```

Comment:
Deprecated `showError` and `cleanError`. Errors will now be displayed on a client-side defined page instead of being sent to the server for rendering.

Reply:
cc @liangfung